mobile-widgets/qmlmanager.cpp

```diff
@@ -1583,6 +1583,18 @@ void QMLManager::setLibdcLog(bool value)
 	emit libdcLogChanged();
 }
 
+bool QMLManager::developer() const
+{
+	return m_developer;
+}
+
+void QMLManager::setDeveloper(bool value)
+{
+	m_developer = value;
+	DCDeviceData::instance()->setSaveLog(value);
+	emit developerChanged();
+}
+
 bool QMLManager::btEnabled() const
```

Comment:
Ah, a little quirk in order for the hide/show of the developer option in the global drawer to function properly. Maybe it can be done with pure QML, didn't spend that long on the subject.